com.unity.perception/Documentation~/FAQ/FAQ.md

```diff
@@ -0,0 +1,584 @@
+  public class PrefabCluster : ScriptableObject
+  {
+      public GameObjectParameter clusterPrefabs;
+  }
+  ```
+
+  We can now create a cluster asset using the ***Assets -> Create -> Test -> PrefabCluster*** menu option and populate its list of prefabs. Each cluster contains one `GameObjectParameter`, which will hold the list of prefabs and provide us with a `Sample()` function.
+
+  To be able to edit these clusters with the same editor UI available for Randomizers, you will also need to add an empty custom editor for the `PrefabCluster` class that extends our bespoke `ParameterUIElementsEditor` class:
+
+  ```C#
+  using UnityEditor;
+  using UnityEditor.Perception.Randomization;
+
+  [CustomEditor(typeof(PrefabCluster))]
+  public class PrefabClusterEditor : ParameterUIElementsEditor { }
+  ```
+
+  Note that any editor scripts must be placed inside a folder named "Editor" within your project. "Editor" is a special folder name in Unity that prevents editor code from compiling into a player during the build process. For example, the file path for the `PrefabClusterEditor` script above could be `.../Assets/Scripts/Editor/PrefabClusterEditor`.
+
+  The ***Inspector*** view of a prefab cluster asset looks like below:
+
+  <p align="center">
+<img src="images/prefab_cluster.png" width="400"/>
+</p>  
+
+  Now all that is left is to use our prefab clusters inside a Randomizer. Here is some sample code:
+
+  ```C#
+  using System;
+using UnityEngine;
+
+[Serializable]
+[UnityEngine.Perception.Randomization.Randomizers.AddRandomizerMenu("My Randomizers/Cluster Randomizer")]
+public class ClusterRandomizer : UnityEngine.Perception.Randomization.Randomizers.Randomizer
+{
+    public PrefabCluster[] clusters;
+    protected override void OnIterationStart()
+    {
+        //select a random prefab from each cluster
+        foreach (var cluster in clusters)
+        {
+            var prefab = cluster.clusterPrefabs.Sample();
+            //do things with this prefab, e.g. create instances of it, etc. 
+        }
+    }
+}
+  ```
+This Randomizer takes a list of `PrefabCluster` assets, then, on each Iteration, it goes through all the provided clusters and samples one prefab from each. The ***Inspector*** view for this Randomizer looks like this:
+
+<p align="center">
+<img src="images/cluster_randomizer.png" width="400"/>
+</p>  
+
+---
+</details>
+
+<details>
+  <summary><strong>Q: How can I specify an exact number of objects to place using the sample foreground object placement Randomizer?</strong> </summary><br>
+
+The provided `ForegroundObjectPlacementRandomizer` uses Poisson Disk sampling to find randomly positioned points in the space denoted by the provided `Width` and `Height` values. The lower bound on the distance between the sampled points will be `Separation Distance`. The number of sampled points will be the maximum number of points in the given area that match these criteria.
+
```

Comment:
Another thing that maybe should be added to the default randomizers

Reply:
👍